src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -500,6 +507,11 @@ private void AutoCompleteCluster(object parameter)
             // Bail out from connect state
             wsViewModel.CancelActiveState();
 
+            if (!CanAutoComplete(null))
+            {
+                return;
+            }
```

Comment:
This currently is already checked as part of the command at https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Commands/PortCommands.cs#L36, right?